packages/bigquery/src/table.js

```diff
@@ -1015,18 +1015,19 @@ Table.prototype.import = function(source, metadata, callback) {
  * //-
  * function insertHandler(err, insertErrors, apiResponse) {
  *   // err (object):
- *   //   An API error occurred.
- *
- *   // insertErrors (object[]):
- *   //   If populated, some rows failed to insert, while others may have
- *   //   succeeded.
- *   //
- *   // insertErrors[].row (original individual row object passed to `insert`)
- *   // insertErrors[].errors[].reason
- *   // insertErrors[].errors[].message
- *
- *   // See https://developers.google.com/bigquery/troubleshooting-errors for
- *   // recommendations on handling errors.
+ *   //   An API error or partial failure occurred.
+ *
+ *   if (err.name === 'PartialFailureError') {
```

Comment:
Well, my thought with the property was to avoid a string comparison, so I don't really see that as being any different than the check against the name. If you think that's the way to go, then LGTM.